### PR TITLE
Add contact and footer

### PR DESCRIPTION
**Pull request by Adetunbosun**
(1) I build the contact form in the footer
(2) I made sure all fields have placeholder text.
(3) I made sure all fields have the correct type.
(4) I made sure there are no empty field 
(5) I collected data  by using the [Formspree service](https://formspree.io/html).
(6) In order to lay out the elements on the page I  use Flexbox.

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @Adetunbosun , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that contains linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -129,5 +129,26 @@ <h4 class="list-title">Skills</h4>
...
   </section>
+  <section class="contact">
+    <h3 class="contact-header">Get started</h3>
+    <p class="contact-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+    <form class="form" action="https://formspree.io/f/mjvzkaal" method="post">
+      <input type="text" maxlength="30" placeholder="Your name" class="form-input" required />
+      <input type="email" placeholder="Your email" class="form-input" required />
+      <textarea maxlength="500" placeholder="Enter your message here..." class="form-text" required></textarea>
+      <button type="submit" class="form-btn">Start collaboration</button>
+    </form>
```
Comment (ha-manel) [inline index.html]: - [ ] When I tried to submit a message I got the following error, kindly fix it. Suggestion: try adding the `name` attribute maybe that will fix it.
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Adetunbosun 👋 
Great job so far, your website looks awesome 💯 There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Adetunbosun , 

# STATUS: APPROVED 👏🏽 👏🏽 👏🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Gitflow ✔️ 
- Professional README file ✔️ 
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**